common/snapshots.py

```diff
@@ -1634,7 +1637,7 @@ def statFreeSpaceSsh(self):
         # /tmp           127266564 115596412   5182296  96% /
         #                                      ^^^^^^^
         for line in output.split(b'\n'):
-            m = re.match(b'^.*?\s+\d+\s+\d+\s+(\d+)\s+\d+%', line, re.M)
+            m = re.match(r'^.*?\s+\d+\s+\d+\s+(\d+)\s+\d+%', line.decode(), re.M)
```

Comment:
The output from shell was catched as binary instead of a string. No need for that.
Converted that to a real string (via ` .decode()`) and run a raw regex string (via `r''`) on it.